apps/emqx_cluster_link/src/emqx_cluster_link_router_syncer.erl

```diff
@@ -191,6 +191,10 @@ start_link_client(Actor, LinkConf) ->
             Error
     end.
 
+stop_link_client(#st{client = ClientPid} = St0) ->
+    ok = emqtt:stop(ClientPid),
+    St0#st{client = undefined}.
```

Comment:
Tiny nit. Since it operates with `#st{}` this function belongs to the section with its siblings (e.g. `handle_client_down/2`).

Reply:
moved closer to `handle_client_down/2`